app/containers/App/reducer.js

```diff
@@ -15,6 +16,7 @@ const initialState = fromJS({
   loggedIn: loggedIn(),
   loginError: false,
   fetchingUser: false,
+  displayNavbarBackground: false,
```

Comment:
Shouldn't the default be `true`?
Or is the plan to use have it transparent on other pages than the splash?

Reply:
I'm not sure what's the best approach here. Right now I've only considered the homepage (meaning, yes, this will currently break on other pages). 

Set this to true and dispatch the redux action to switch the background on mount?

Reply:
Reading the second bullet point [here](https://github.com/brigade/react-waypoint#usage) it sounds like you wouldn't even have to fire it manually.

Reply:
ah that's true. cool! u are so smart. this is why i want people to review my prs 🤓

Reply:
works like a charm. fixed!